app/views/contexts/section_summary_context.py

```diff
@@ -21,10 +24,16 @@ def __call__(self, current_location, return_to="section-summary"):
             if isinstance(title_for_location, dict)
             else title_for_location
         )
+
+        page_title = self._schema.get_custom_page_title_for_section(
+            current_location.section_id
+        )
+
         return {
             "summary": {
                 "title": title,
-                "page_title": self._get_safe_page_title(title_for_location),
+                "page_title": page_title
+                or self._get_safe_page_title(title_for_location),
```

Comment:
I think it would be clearer if this was done above where `page_title` is assigned.

Reply:
Done

Reply:
Done